### PR TITLE
:zap: 11.0 added synchronization among the same POS

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Comment by yelizariev** [comment]
It's a Feature, not an Improvement. Use appropriate commit tag

**Comment by yelizariev** [comment]
Also, update version

**Review by yelizariev — APPROVED** [review]
Looks good for me

File: pos_disable_payment/static/src/js/pos_disable_payment.js [hunk 1]
```diff
@@ -19,8 +19,11 @@ odoo.define('pos_disable_payment', function(require){
     var PosModelSuper = models.PosModel;
     models.PosModel = models.PosModel.extend({
         set_cashier: function(){
+            var old_cashier_id = this.db.get_cashier() && this.db.get_cashier().id;
             PosModelSuper.prototype.set_cashier.apply(this, arguments);
-            this.trigger('change:cashier',this);
+            if (old_cashier_id !== this.db.get_cashier().id) {
+                this.trigger('change:cashier', this);
```
Comment (yelizariev) [inline pos_disable_payment/static/src/js/pos_disable_payment.js]: ok

**Review by yelizariev — APPROVED** [review]

File: pos_disable_payment/doc/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Updates
 =======
 
+`2.5.1`
+-------
+- FIX: Maximum call stack size exceeded due to pos_session_id in db is assigned later than that func exceeds call stack
```
Comment (yelizariev) [inline pos_disable_payment/doc/changelog.rst]: It's too technical. You must says when the error happens

**Comment by KolushovAlexandr** [comment]
the same will be merged with the branch 10.0-ms-sync-conflicts

**Pull request closed without merging**